CFC_WebApp/api/cfc_webapp.py

```diff
@@ -14,8 +14,8 @@
 import urllib
 import requests
 # For decoding JWTs on the client side
-import oauth2client.client
-from oauth2client.crypt import AppIdentityError
+#import oauth2client.client
```

Comment:
This was part of the patch that I sent you. It is only intended to make development easier.
Please don't check it in.